### DISCUSSION TITLE
Change menu highlight color

### DISCUSSION THREAD

**Discussion by rxboy1983** [valinet/ExplorerPatcher#199] [Show and tell]
For change color of highlight in menu:
Edit in regedit: 
HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Accent - AccentPalette.
Also can use app AccentPaletteTool. 

Tested and work on Dev build 22483.1011.

Also related: https://github.com/valinet/ExplorerPatcher/issues/18

**Comment by valinet** [comment]
Thanks. Awesome discovery.

**Reply by Anixx** [reply]
![изображение](https://user-images.githubusercontent.com/1271736/138686785-3c71f05a-f9f1-41ab-b0db-19ca46484d1b.png)

`
explorer shell:::{ED834ED6-4B5A-4bfe-8F11-A626DCB6A921} -Microsoft.Personalization\pageColorization`

**Comment by rxboy1983** [comment]
**Anixx**,
Run>explorer shell:::{ED834ED6-4B5A-4bfe-8F11-A626DCB6A921} -Microsoft.Personalization\pageColorization
- here can't get separate colors for Menu Start, Taskbar, items in Settings, Menu Center, Calendar, Notifications, items in Microsoft Store, highlight in menu, selected/pushed Start button, pushed buttons in taskbar, taskband etc.. Settings colorised everything.

Editing AccentPalette is more configurable. Also can get really black background for Start Menu. :)

**Comment by rxboy1983** [comment]
![Zrzut ekranu 2021-10-25 192510](https://user-images.githubusercontent.com/93114065/138742049-b252a6dd-70bc-4d12-a3d3-94a3e6395568.png)
Edit it to change highlight color.

**Comment by Poopooracoocoo** [comment]
Does Accent Applicator happen to work? Haven't used that in ages.
